galaxy_ng/app/api/v3/viewsets/collection.py

```diff
@@ -274,8 +275,8 @@ def get(self, request, *args, **kwargs):
             )
 
         metrics.collection_artifact_download_failures.labels(status=response.status_code).inc()
-        raise APIException('Unexpected response from content app. '
-                           f'Code: {response.status_code}.')
+        raise APIException(_('Unexpected response from content app. '
+                           f'Code: {response.status_code}.'))
```

Comment:
this has an f-string which should be removed, looks like ci didn't catch it because it is on multiple lines